scripts/sandbox/generate_circle_path_test.R

```diff
@@ -0,0 +1,151 @@
+num_pies <- 2000
+
+x_rng <- c(0, 150)
+y_rng <- c(0, 180)
+view_coords <- c(300, 200)
+set.seed(99)
+x_coords <- runif(n = num_pies, min = x_rng[1], max = x_rng[2])
+y_coords <- runif(n = num_pies, min = y_rng[1], max = y_rng[2])
+
+r_0 <- rlnorm(num_pies, 0, .6)/2
+r_1 <- rlnorm(num_pies, 0, .6)/2
+
+library(xml2)
+library(dplyr)
+m = xml_new_document() %>% 
+  xml_add_child('html', prefix="og: http://ogp.me/ns#", lang="en") 
+m %>% xml_add_child('head') %>% 
+  xml_add_child('script', src="https://d3js.org/d3.v4.min.js", charset="utf-8") %>% 
+  xml2::xml_add_child('dummy') # trailing solidus hack?
+bd <- m %>% xml_add_child('body')
+bd %>% xml_add_child('style',
+                     ".therm {
+			fill:#b2e7e2;
+      stroke:#b2e7e2;
+      stroke-width:0.3;
+			fill-opacity:0.5;
+                     }
+    .dot {
+                     stroke-width:0.3;
+                     fill-opacity:0.5;
+                     }
+		.irr {
+			fill:red;
+      stroke:red;
+      stroke-width:0.3;
+			fill-opacity:0.8;
+		}")
+svg <- bd %>% 
+  xml_add_child('svg', xmlns="http://www.w3.org/2000/svg", 
+                "xmlns:xlink"="http://www.w3.org/1999/xlink", version="1.1", preserveAspectRatio="xMinYMin meet", 
+                viewBox=sprintf("0 0 %s %s", view_coords[1], view_coords[2]))
+
+path_template <- "M%1.1f %1.1f a %1.1f %1.1f 0 1 1 0 0.01 " #x start, y-start, radius, radius
```

Comment:
This is how the `d` values for a path are generated. 

The pattern is creating json w/ an element for each category. But, there is a ton of redundant information and we could collapse this down if we thought the json was too big - we really just need x,y,radius and could build the json in the browser if needed. But I think that gzipping would take care of those redundant pieces of info anyhow so this wouldn't be necessary.